### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by carsten19**
- central scene command class for channel naming
- network layout improvements: better overview and colors based on hops from controller
- preinstall: add -u option to unzip for Mac and add missing ldconfig call
Version increase

### CODE REVIEW

**Comment by AlCalzone** [comment]
This change requires `ldconfig` to be allowed in
https://github.com/ioBroker/ioBroker/blob/master/installer.sh and
https://github.com/ioBroker/ioBroker/blob/master/fix_installation.sh

File: admin/js/comClasses.js [hunk 1]
```diff
@@ -217,7 +217,8 @@ var comclasses = {
     0x2F: {name: 'ZIP_ADV_SERVICES',                       role: ''},
     0x2e: {name: 'ZIP_CLIENT',                             role: ''},
     0x24: {name: 'ZIP_SERVER',                             role: ''},
-    0x23: {name: 'ZIP_SERVICES',                           role: ''}
+    0x23: {name: 'ZIP_SERVICES',                           role: ''},
+    0x5B: {name: 'CENTRAL_SCENE',                          role: ''}
```
Comment (AlCalzone) [inline admin/js/comClasses.js]: We might want to double check if there is a definition for all command classes, see 
https://github.com/AlCalzone/node-zwave-js/blob/master/src/lib/commandclass/CommandClasses.ts#L4 for reference

**Comment by carsten19** [comment]
> This change requires `ldconfig` to be allowed in
> https://github.com/ioBroker/ioBroker/blob/master/installer.sh and
> https://github.com/ioBroker/ioBroker/blob/master/fix_installation.sh

I fear this will create a lot of issues when the people don't know that the fixer is a prerequisite.
On the other side this missing ldconfig call is an issue as the installation doesn't work without. Isn't there a chance to prompt for password?

**Comment by AlCalzone** [comment]
>  Isn't there a chance to prompt for password?

No. I wouldn't know a foolsafe way to forward such a prompt to the admin UI.

The better approach would be https://github.com/ioBroker/ioBroker.js-controller/issues/348 IMO - which means that each adapter specifies what root commands it needs. The js-controller would have to allow those (manage permissions) and then `sudo ldconfig` would be no issue.

File: admin/js/comClasses.js (hunk 1 above)
Comment (carsten19) [reply admin/js/comClasses.js]: Thanks, remaining com classes added.

**Comment by carsten19** [comment]
Reverted ldconfig change for now

**Comment by carsten19** [comment]
Ready from my side, the travis errors are just the usual strange host not found issues

**Comment by carsten19** [comment]
@AlCalzone or @Apollon77 can you merge if you are fine with that change please?

**Review by AlCalzone — APPROVED** [review]
LGTM

**Comment by AlCalzone** [comment]
I squash-merged, so please make sure to use a new branch for your next changes or reset it to the current master

**Pull request merged**